src/uct/api/uct.h

```diff
@@ -1788,9 +1789,10 @@ UCT_INLINE_API ucs_status_t uct_ep_tag_eager_short(uct_ep_h ep, uct_tag_t tag,
 
 /**
  * @ingroup UCT_TAG
- * @brief Send tagged message using bcopy eager protocol.
+ * @brief Bcopy eager tagged-send operation.
  *
- * Eager protocol means that the whole data is sent immediately to the peer.
+ * This routine sends a message using buffered copy eager protocol. Eager
```

Comment:
I suggest to elaborate on buffered. From the documentation one may think there is no difference between buffered and short. I would add that the user data is cached in the bouncing buffer and then later the data is sent from the buffer. Also please describe the motivation  - a.k.a the bouncing buffer is allocated given network constrains and ready for immediate utilization by HW.